src/styles/base/_typography.scss

```diff
@@ -1,5 +1,5 @@
 // Robot Condensed:
-@import url(https://fonts.googleapis.com/css?family=Roboto:300,400,500);
+@import url(https://fonts.googleapis.com/css?family=Roboto:300,400,500,700);
```

Comment:
This is a lot of fonts and a lot of weights. Are we sure we need effectively 7 font varieties? It might also make it hard for future develoeprs who are unsure what to use, as well as taking up a lot of bandwidth. Anyway, these should be condensed as much as possible, you might be able to request them all in one request.